components/playground/main.go

```diff
@@ -73,23 +73,27 @@ func execute() error {
 	pdNum := 1
 	host := "127.0.0.1"
 	monitor := false
+	dbConfigPath := ""
+	kvConfigPath := ""
+	pdConfigPath := ""
 
 	rootCmd := &cobra.Command{
 		Use: "tiup playground [version]",
 		Long: `Bootstrap a TiDB cluster in your local host, the latest release version will be chosen
 if you don't specified a version.
 
 Examples:
-  $ tiup playground nightly                         # Start a TiDB nightly version local cluster
-  $ tiup playground v3.0.10 --db 3 --pd 3 --kv 3    # Start a local cluster with 10 nodes
-  $ tiup playground nightly --monitor               # Start a local cluster with monitor system`,
+  $ tiup playground nightly                                                           # Start a TiDB nightly version local cluster
+  $ tiup playground v3.0.10 --db 3 --pd 3 --kv 3                                      # Start a local cluster with 10 nodes
+  $ tiup playground nightly --monitor                                                 # Start a local cluster with monitor system
+  $ tiup playground --pd.config ./pd.toml --db.config ./db.toml --kv.config ./kv.toml # Start a local cluster with specified configuration file`,
```

Comment:
I think we don't need to pass all config flags in examples. e.g
```
tiup playground --pd.config ./pd.tom
```

Reply:
Fixed.